common/types/snapshot.ts

```diff
@@ -5,7 +5,7 @@ export const snapshotSchema = z.object( {
 	atomicSiteId: z.number(),
 	localSiteId: z.string(),
 	date: z.number(),
-	name: z.string(),
+	name: z.string().optional(),
```

Comment:
```suggestion
	name: z.string().default( '' ),
```
A suggestion for future similar changes is to use zod's `default` method. It's not always the right choice, but in cases like this, where we always (or mostly, at least) default to an empty string anyway, it just makes things easier.